libs/frontend/feature-build/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -79,6 +82,16 @@ const renderImageHover = (
     )
   }
 
+  // Pass the margin of img to the wrapper element, and set the margin of the rendered img
+  // to 0 during hover to avoid shifting issue.
+  const imgStyleProp = structuredClone(props.style ?? {}) as Record<string, unknown>
+  imgStyleProp['margin'] = '0'
+
+  const img = h(component.tag as string, {
+    ...props,
+    style: imgStyleProp,
+  })
```

Comment:
Hi @sampullman 

It seems that only `margin` would lead to the shift issue we've observed in the editor, while `padding` will not.
So I only pass the `margin` of `img` to the wrapper element here, which seems to fix the issue.

Reply:
Got it!